AABB_tree/doc/AABB_tree/Concepts/AABBRayIntersectionTraits.h

```diff
@@ -34,9 +73,9 @@ class AABBRayIntersectionTraits {
     respectively.
 
     Provides the operators:
-    `std::optional<FT> operator()(const Ray_3& r, const Bounding_box& bbox)`.
-    `std::optional<std::pair<FT, Intersection_and_primitive_id<Ray_3>::%Type > >
-     operator()(const Ray_3& r, const Primitive& primitive)`.
+    `std::optional<FT> operator()(const Ray& r, const Bounding_box& bbox)`.
```

Comment:
As it refines `AABBTraits`  it is defined.

Reply:
I meant, usually I redefine the types coming from the base concepts so that there is a link.

Maybe `typedef AABBTraits::Bounding_box Bounding_box`? (This would be clearer if we inherited concepts!)